ckanext/harvest/harvesters/ckanharvester.py

```diff
@@ -1,8 +1,11 @@
-import urllib
-import urllib2
+import requests
+from requests.exceptions import HTTPError
```

Comment:
`HTTPError` and `InvalidURL` both inherit from `RequestException `, it might be better to just catch that.

Reply:
I don't believe `httplib.HTTPException` or `socket.error` will be raised when using `requests`, as they are specific variations of the `RequestException ` that is used by `requests`.

Reply:
@thomasleese good point. I was just porting the changes avoiding as much changes as possible (to make the merge upstream easier), but yeah.. it makes sense